lib/radar.js

```diff
@@ -31,29 +35,27 @@ const request = (north, west, south, east, when, opt = {}) => {
 	opt = Object.assign({}, defaults, opt)
 
 	const query = {
-		bounds: [north, south, west, east].join(','),
-		callback: 'jsonp',
-		// options
-		faa: opt.FAA ? '1' : '0',
-		flarm: opt.FLARM ? '1' : '0',
-		mlat: opt.MLAT ? '1' : '0',
-		adsb: opt.ADSB ? '1' : '0',
-		air: opt.inAir ? '1' : '0',
-		gnd: opt.onGround ? '1' : '0',
-		vehicles: opt.inactive ? '1' : '0',
-		gliders: opt.gliders ? '1' : '0'
-		// todo: bounds, estimated, maxage, stats, history, prefetch
+		bounds: [north, south, west, east].join(','),	// 
+		callback: 'jsonp',						// 
+		// options: altitude, estimated, maxage, speed, stats, ems, (history, prefetch)?
+		// ToDo: 	maxage, stats, (history, prefetch)?
+		faa: opt.FAA ? '1' : '0',				// 
+		mlat: opt.MLAT ? '1' : '0',				// 
+		flarm: opt.FLARM ? '1' : '0',			// 
+		adsb: opt.ADSB ? '1' : '0',				// 
+		gnd: opt.onGround ? '1' : '0',			// 
+		air: opt.inAir ? '1' : '0',				// 
+		vehicles: opt.inactive ? '1' : '0',		// 
+		estimated: opt.ESTIMATED ? '1' : '0',	// 
+		//maxage: opt.maxage ? '1' : '0', 		// The Max time in [300-14400 sec] for ESTIMATED aircraft
+		gliders: opt.gliders ? '1' : '0'		// 
+		//stats: opt.stats ? '1' : '0',			// ?
+		//ems: opt.ems ? '1' : '0',				// ?
 	}
 	if (when) query.history = Math.round(when / 1000)
 
 	const url = endpoint + '?' + qs.stringify(query)
-	return fetch(url, {
-		mode: 'cors',
-		redirect: 'follow',
-		headers,
-		referrer: 'no-referrer',
-		referrerPolicy: 'no-referrer'
-	})
+	return fetch(url, {mode: 'cors', redirect: 'follow', headers})
```

Comment:
There is a reason why this is in place. Keep it as before and propose changes in another PR.